pkg/ingress/ingress_test.go

```diff
@@ -1 +1,241 @@
 package ingress
+
+import (
+	"testing"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	operatorv1 "github.com/openshift/api/operator/v1"
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func TestIngressPull(t *testing.T) {
+	testCases := []struct {
+		ingressName         string
+		ingressNamespace    string
+		expectedError       bool
+		addToRuntimeObjects bool
+		expectedErrorText   string
+	}{
+		{
+			expectedError:       false,
+			addToRuntimeObjects: true,
+			expectedErrorText:   "",
+			ingressName:         "test",
+			ingressNamespace:    "test",
+		},
+		{
+			expectedError:       true,
+			addToRuntimeObjects: false,
+			expectedErrorText:   "ingresscontroller object test not found in namespace test",
+			ingressName:         "test",
+			ingressNamespace:    "test",
+		},
+		{
+			expectedError:       true,
+			addToRuntimeObjects: true,
+			expectedErrorText:   "ingresscontroller object test not found in namespace test",
```

Comment:
It looks like error is invalid. It has to be "The` ingresscontroller name is empty" no?

Reply:
Fixed.